### PR TITLE
[8.0] Add accout invoice merge payment module

### PR DESCRIPTION
**Pull request by adrienpeiffer**
Add accout invoice merge payment module that was written to extend the functionality of Account Invoice Merge module to support fields added in Account Payment Partner from OCA/bank-payment.

### CODE REVIEW

File: account_invoice_merge_payment/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    'author': "ACSONE SA/NV,Odoo Community Association (OCA)",
+    'website': "http://acsone.eu",
+    'category': 'Invoicing & Payments',
+    'version': '0.1',
+    'license': 'AGPL-3',
+    'depends': [
+        'account_invoice_merge',
+        'account_payment_partner',
+    ],
+    'data': [],
```
Comment (Cedric-Pigeon) [inline account_invoice_merge_payment/__openerp__.py]: I suggest to remove empty section.

File: account_invoice_merge_payment/README.rst [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+============
+
+To install this module, you need to:
+
+ * Click on install button
+
+For further information, please visit:
+
+ * https://www.odoo.com/forum/help-1
+
```
Comment (sbidoul) [inline account_invoice_merge_payment/README.rst:14]: I'd update this section to simply say the module auto installs when account_invoice_merge and account_payment_partner are installed.

**Comment by sbidoul** [comment]
A small remark in README, otherwise :+1:

**Comment by Cedric-Pigeon** [comment]
A small remark in manifest, otherwise :+1:

**Comment by adrienpeiffer** [comment]
@sbidoul  @Cedric-Pigeon Thanks for your reviews

**Comment by sbidoul** [comment]
This module is important to make the account_invoice_merge work correctly without losing data in presence of OCA/bank-payment modules. As it does not add functionality but merly make thing work correctly it auto-installs when it's dependencies are installed.

:+1: (code review and functional test)

File: account_invoice_merge_payment/README.rst [hunk 3]
```diff
@@ -0,0 +1,32 @@
+Account Invoice Merge Payment
```
Comment (lmignon) [inline account_invoice_merge_payment/README.rst:4]: licence badge missing

``` rest
.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
    :alt: License AGPL-3
```

**Comment by lmignon** [comment]
A small remark in README, otherwise ￼:+1: (code review)

**Comment by adrienpeiffer** [comment]
@lmignon I added the badge. Thanks !

**Comment by lmignon** [comment]
@adrienpeiffer Thank you :+1:

**Comment by sbidoul** [comment]
@adrienpeiffer can you rebase (confllict in travis.yml)?

**Comment by adrienpeiffer** [comment]
@sbidoul It's done !

**Comment by sbidoul** [comment]
I added OCA/bank-payment to runbot and restarted the build.

**Comment by adrienpeiffer** [comment]
Thanks @sbidoul. Now CI are green !

**Comment by adrienpeiffer** [comment]
I think this one is ready to merge

**Pull request merged**